### PR TITLE
CLD-657: updated readme for docker release 11.0.0-centos-1.0.1

### PR DESCRIPTION
**Pull request by barkhachoithani**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,7 +47,7 @@ Supported Docker architectures: x86_64
 
 Base OS: CentOS
 
-Latest supported MarkLogic Server version: 10.0-9.5
+Latest supported MarkLogic Server version: 11.0.0
```
Comment (jkerr5) [inline README.md:50]: Can we just remove this line so we don't have to update this for every release? Is there value to provide this note? Users can see it in the tags, right? Perhaps we can add the "Recent Tags" card to our docker hub page to show the tags that have most recently been updated.

File: README.md [hunk 2]
```diff
@@ -28,15 +28,16 @@ This README serves as a technical guide for using MarkLogic Docker and MarkLogic
...
 - Examples in this document use Docker Engine and Docker CLI to create and manage containers. Follow the documentation for instructions on how to install Docker: see Docker Engine (https://docs.docker.com/engine/)
 - To access the MarkLogic Admin interface and App Servers in our examples, you need a desktop browser. See "Supported Browsers" in the [support matrix](https://developer.marklogic.com/products/support-matrix/) for a list of supported browsers.
 
 # Supported tags
 
 Note: MarkLogic Server Docker images follow a specific tagging format: `{ML release version}-{platform}-{ML Docker release version}`
 
-- 10.0-9.5-centos-1.0.0 - This current release of the MarkLogic Server Developer Docker image includes all features and is limited to developer use
-- [Older Supported Tags](#older-supported-tags)
+- 11.0.0-centos-1.0.1 - This current release of the MarkLogic Server Docker image includes all the features.
```
Comment (jkerr5) [inline README.md:39]: Same comment as below

**Review by jkerr5 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (barkhachoithani) [reply README.md:50]: Currently, there is no value to display the version in the readme.

File: README.md (hunk 2 above)
Comment (barkhachoithani) [reply README.md]: removed current version info.

**Review by jkerr5 — APPROVED** [review]

**Pull request merged**